crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -103,17 +103,18 @@ impl<S: SlotData, B, T> SlotCommit<S, B, T> {
 
 impl LedgerDB {
     /// Open a [`LedgerDB`] (backed by RocksDB) at the specified path.
-    /// The returned instance will be at the path `{path}/ledger-db`.
+    /// Will take optional column families, used for migration purposes.
+    /// The returned instance will be at the path `{path}/ledger`.
     #[instrument(level = "trace", skip_all, err)]
-    pub fn with_config(cfg: &RocksdbConfig) -> Result<Self, anyhow::Error> {
+    pub fn with_config(
+        cfg: &RocksdbConfig,
+        column_families: Option<Vec<String>>,
```

Comment:
I believe column families are rocksdb related. Please include them in `RocksdbConfig`

Reply:
On second thought, I am not sure if it should be in the config, because we can already get it with a function just by giving the path

Reply:
I'll think on this a bit more

Reply:
It also kinda makes sense as well :D

Reply:
Okay, so I think let's skip that one for now because the column_families should be optional and if it is None it should be either LEDGER_TABLES STATE_TABLES or NATIVE_STATE_TABLES, to extract which to use we can use the path or pass another parameter but I feel like that's just extra code

I recommend implementing this if we really need to

WDYT?

Reply:
I am not sure i follow. In either way, we are passing `column_families` here... why not move this param to `RocksdbConfig` and use it the exact same way you do in the PR?

Reply:
I can do that for ledger db but for example here I will need to create two configs just for that, there are other examples to this as well, I think this will just generate more code + all of the column families are accessible publicly by all crates importing sov-db as they are pub consts
<img width="882" alt="Screenshot 2024-12-05 at 22 48 48" src="https://github.com/user-attachments/assets/d3c8e47b-79be-437a-b4a9-675af2198ec5">

Reply:
In this example, we pass `None` and then `unwrap_or_else` when the default list of CFs are needed.